protected/components/views/patientDiagnosesAndMedicationsWidget.php

```diff
@@ -24,7 +24,7 @@
                             <?php foreach ($this->patient->secondarydiagnoses as $diagnosis): ?>
                                 <tr>
                                     <td><?php echo $diagnosis->disorder->fully_specified_name; ?></td>
-                                    <td><?php echo ($diagnosis->is_confirmed !== 0) ? 'Confirmed' : 'Unconfirmed'; ?></td>
+                                    <td><?php echo ($diagnosis->is_confirmed != 0) ? 'Confirmed' : 'Unconfirmed'; ?></td>
```

Comment:
We will need to revisit this logic in a future revision. I suspect what will happen is that legacy patients will be marked as unconfirmed despite being actually confirmed. This will require a significant change to the modelling so I'll get Liam to take care of that.